app/modules/intelligence/agents_copy/plan.py

```
@@ -0,0 +1,145 @@
+        pass
+
+    def delete_agent(self, agent_id: str) -> None:
+        """Delete an agent"""
+        pass
+
+
+class AgentContext(BaseModel):
+    """Context for building an agent"""
+
+    message: str
+    llm_config: LLMConfig
```

Ruff (0.8.2)

Undefined name `LLMConfig`

(F821)